exercises/pangram/example.rs

```diff
@@ -0,0 +1,18 @@
+use std::collections::BTreeSet;
+use std::ascii::AsciiExt;
+use std::iter::FromIterator;
+
+pub fn is_pangram(sentence: &str) -> bool {
+    sentence.to_lowercase()
+            .chars()
```

Comment:
The `.graphemes` iterator might be a better choice here:

http://www.steveklabnik.com/rust-issue-17340/#indexing-strings